site/update-image.md

```diff
@@ -1,7 +1,15 @@
 # Update Image
 
-Use the `update` command to apply patches to an existing WebLogic Docker image. The required options for the command
-are marked with an asterisk (*). The password can be provided in one of the three ways:
+Once you have created a Docker image with the Image Tool, you may want to change it from time to time.  The `update` 
```

Comment:
Once -> After (once – do not use to mean after; use only to mean less than twice.)